js/templates/pageConnectModal.html

```diff
@@ -0,0 +1,16 @@
+<div <% ob.mode && print(`class="mode-${ob.mode}"`) %>>
```

Comment:
@morebrownies The template for this modal is fairly generic and abstracted. I wanted to make it so it was re-usable to any page we wanted to give this connecting modal treatment to. The individual page views will pass in a config that specify the particular messaging the connecting modal should use when connecting to that page.

As of this PR, I've only applied it to the user page (either via guid or handle) and the listing detail page.